packages/slap/slap.3.0.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "abe@sf.ecei.tohoku.ac.jp"
+authors: [ "Akinori ABE <abe@sf.ecei.tohoku.ac.jp>" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "http://akabe.github.io/slap/"
+dev-repo: "https://github.com/akabe/slap.git"
+bug-reports: "https://github.com/akabe/slap/issues"
+build: [
+  ["ocaml" "setup.ml" "-configure"
+   "--prefix" prefix
+   "--disable-ppx"  {ocaml-version < "4.02"}]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "slap"]
```

Comment:
Unless I am mistaken, you install an executable `ppx_slap` so the above will not be enough to properly remove the package.  IMHO, you should update your tarball and resubmit a PR for to replace this version.